server/server.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/rawdaGastan/grid3_auto_deployer/internal"
+	"github.com/rawdaGastan/grid3_auto_deployer/middlewares"
+	"github.com/rawdaGastan/grid3_auto_deployer/models"
+	"github.com/rawdaGastan/grid3_auto_deployer/routes"
+)
+
+type Server struct {
+}
+
+func NewServer(dbFile string) (server Server, err error) {
+	content, err := internal.ReadFile("./.env")
+	if err != nil {
+		return
+	}
+
+	secret, err := internal.ParseEnv(content)
+	if err != nil {
+		return
+	}
+
+	db := models.NewDB()
+	err = db.Connect(dbFile)
+	if err != nil {
+		return
+	}
+	err = db.Migrate()
+	if err != nil {
+		return
+	}
+
+	router := routes.NewRouter(secret, db)
+	r := mux.NewRouter()
+	r.Use(middlewares.Middleware)
+	r.HandleFunc("/signup", router.SignUpHandler).Methods("POST")
+	r.HandleFunc("/verify", router.VerifyUser).Methods("POST")
+	r.HandleFunc("/signin", router.SignInHandler).Methods("GET")
+	r.HandleFunc("/home", router.Home).Methods("GET")
+	r.HandleFunc("/refresh", router.RefreshJWT).Methods("GET")
+	r.HandleFunc("/logout", router.Logout).Methods("GET")
+	r.HandleFunc("/forgotPassword", router.ForgotPasswordHandler).Methods("GET")
+	r.HandleFunc("/verifycode", router.VerifyCode).Methods("POST")
+	r.HandleFunc("/changePassword", router.ChangePassword).Methods("POST")
+	r.HandleFunc("/updateAccount", router.UpdateAccount).Methods("POST")
+	r.HandleFunc("/getUser", router.GetUser).Methods("GET")
```

Comment:
"/getUser" -> /user/get 
note: add params i discussed in comments for the needed handlers